src/ipc/Kconfig

```diff
@@ -23,4 +23,12 @@ config IPC_MAJOR_4
 
 endchoice
 
+config STOP_FOR_PAUSE
+	bool
+	default BAYTRAIL || CHERRYTRAIL || HASWELL || BROADWELL
```

Comment:
Can we make this default for all atm to test the PR. It's probably going to be easier to test/fix on TGL than older HW.